tss2-dlopen/tss2-dlopen-esys.c

```diff
@@ -0,0 +1,2201 @@
+    TSS2_RC (*sym)(ESYS_CONTEXT *ctx, type1, type2, type3, type4, type5, type6, type7, type8, \
+                                      type9, type10, type11) = NULL; \
+    if (!sym) \
+        dlsym(esys_dlhandle, str(fun)); \
+    if (!sym) { \
+        WARN("Function " str(fun) " not found."); \
+        return TSS2_ESYS_RC_NOT_IMPLEMENTED; \
+    } \
+    return sym(ctx, parm1, parm2, parm3, parm4, parm5, parm6, parm7, parm8, \
+                    parm9, parm10, parm11); \
+}
+
+#define MAKE_ESYS_12(fun, type1,parm1, type2,parm2, type3,parm3, type4,parm4, \
+                          type5,parm5, type6,parm6, type7,parm7, type8,parm8, \
+                          type9,parm9, type10,parm10, type11,parm11, type12,parm12) \
+TSS2_RC fun (ESYS_CONTEXT *ctx, type1 parm1, type2 parm2, type3 parm3, type4 parm4, \
+                                type5 parm5, type6 parm6, type7 parm7, type8 parm8, \
+                                type9 parm9, type10 parm10, type11 parm11, type12 parm12) { \
+    TSS2_RC (*sym)(ESYS_CONTEXT *ctx, type1, type2, type3, type4, type5, type6, type7, type8, \
+                                      type9, type10, type11, type12) = NULL; \
+    if (!sym) \
+        dlsym(esys_dlhandle, str(fun)); \
+    if (!sym) { \
+        WARN("Function " str(fun) " not found."); \
+        return TSS2_ESYS_RC_NOT_IMPLEMENTED; \
+    } \
+    return sym(ctx, parm1, parm2, parm3, parm4, parm5, parm6, parm7, parm8, \
+                    parm9, parm10, parm11, parm12); \
+}
+
+#define MAKE_ESYS_13(fun, type1,parm1, type2,parm2, type3,parm3, type4,parm4, \
+                          type5,parm5, type6,parm6, type7,parm7, type8,parm8, \
+                          type9,parm9, type10,parm10, type11,parm11, type12,parm12, \
+                          type13,parm13) \
+TSS2_RC fun (ESYS_CONTEXT *ctx, type1 parm1, type2 parm2, type3 parm3, type4 parm4, \
+                                type5 parm5, type6 parm6, type7 parm7, type8 parm8, \
+                                type9 parm9, type10 parm10, type11 parm11, type12 parm12, \
+                                type13 parm13) { \
+    TSS2_RC (*sym)(ESYS_CONTEXT *ctx, type1, type2, type3, type4, type5, type6, type7, type8, \
+                                      type9, type10, type11, type12, type13) = NULL; \
+    if (!sym) \
+        dlsym(esys_dlhandle, str(fun)); \
+    if (!sym) { \
+        WARN("Function " str(fun) " not found."); \
+        return TSS2_ESYS_RC_NOT_IMPLEMENTED; \
+    } \
+    return sym(ctx, parm1, parm2, parm3, parm4, parm5, parm6, parm7, parm8, \
+                    parm9, parm10, parm11, parm12, parm13); \
+}
+
+
+
+MAKE_ESYS_1(Esys_GetTcti,
+    TSS2_TCTI_CONTEXT **,tcti);
+MAKE_ESYS_2(Esys_GetPollHandles,
+    TSS2_TCTI_POLL_HANDLE **,handles,
+    size_t *,count);
+MAKE_ESYS_1(Esys_SetTimeout,
+    int32_t, timeout);
+MAKE_ESYS_3(Esys_TR_Serialize,
+    ESYS_TR, object,
+    uint8_t **,buffer,
```

Comment:
same here, sometimes you add spaces, sometimes you don't - please make consistent.